public/app/percona/inventory/Inventory.service.ts

```diff
@@ -18,24 +17,23 @@ const BASE_URL = `/v1/inventory`;
 
 export const InventoryService = {
   getAgents(serviceId: string | undefined, nodeId: string | undefined, token?: CancelToken) {
-    return api.post<ServiceAgentListPayload, object>(
-      '/v1/management/Agent/List',
-      { service_id: serviceId, node_id: nodeId },
-      false,
-      token
-    );
+    return api.get<ServiceAgentListPayload, object>('/v1/management/agents', false, {
+      cancelToken: token,
+      params: {
+        service_id: serviceId,
+        node_id: nodeId,
+      },
+    });
   },
-  removeAgent(body: RemoveAgentBody, token?: CancelToken) {
-    return api.post<void, object>(`${BASE_URL}/Agents/Remove`, body, false, token);
+  removeAgent(agentId: string, forceMode = false, token?: CancelToken) {
```

Comment:
Please send in the `forceMode` parameter to the BE (might a be query parameter now?) 

And remove comment :)